src/Card/Card.js

```diff
@@ -1,10 +1,15 @@
+import React from 'react';
+
 import Pane from '../Pane';
-import createWithComponent from '../utils/createWithComponent';
 
-const Card = createWithComponent(Pane, 'div', {
-  defaultProps: {
-    borderRadius: 'large'
-  }
-});
+const Card = ({ children, element, ...props }) => (
+  <Pane element={element} {...props}>
+    {children}
```

Comment:
I don't know if this would be "better", but could this have been `<Pane {...props} />` without desctructing children above?